controllers/books.js

```diff
@@ -1,46 +1,48 @@
-const { bookDB } = require('../db');
+const { bookRepository } = require('../repositories');
+
+const addBook = (req, res) => {
+  const { author, title, pages } = req.body;
+
+  bookRepository.addBookWithAuthorTitleAndPages(author, title, pages);
 
-const addBookEntry = (req, res) => {
-  const newBookEntry = req.body;
-  bookDB.push(newBookEntry);
   res.send('record creation sucessful').status(201);
 };
 
 const getAllBooks = (req, res) => {
```

Comment:
it's more common to use function declaration
[explanation](https://javascript.info/function-expressions#function-expression-vs-function-declaration)